mobile/src/bundles/common/components/checkbox/styles.tsx

```diff
@@ -0,0 +1,9 @@
+import { StyleSheet } from 'react-native';
+
+const styles = StyleSheet.create({
+    container: {
+        marginLeft: -7,
```

Comment:
Why -7?

Reply:
Let's use containerStyle prop for this and pass this if needed,

Reply:
Because by default, the component has a default margin that I can't remove, and by design, it should be below the text on top.

Reply:
I did it through containerStyle